bases/shared/instagoric-server/server.js

```diff
@@ -541,15 +542,16 @@ const startFaucetWorker = async () => {
           }
           break;
         }
+        case 'delegate':
         case COMMANDS["SEND_BLD/IBC"]: {
           [exitCode, txHash] = await sendFunds(address, DELEGATE_AMOUNT);
           break;
         }
+        case 'delegate':
         case COMMANDS["FUND_PROV_POOL"]: {
```

Comment:
We have two `case:delegate`. One on 545 and the other one on 550. Probably it can cause unexpected behavior as only the second 'delegate' case will ever be reached.

Reply:
Let me verify. delegate is actually a backward compatible command which should not affect the faucet flow.
But you are right. Our tests passed because the flow triggered from test is `SEND_BLD/IBC`. But let me verify and fix that to avoid any future discrepancy. Thanks for pointing out.

Reply:
So second `delegate` is redundant. We always want to get first `delegate` case executed (which is happening).
Here's a previous flow screenshot:
 
<img width="611" alt="Screenshot 2024-09-25 at 12 20 47 PM" src="https://github.com/user-attachments/assets/4c1c1953-f0ea-4a4f-89a1-27b64ed12bc0">

Reply:
So I'll remove the redundant case in any follow-up PR. But no flow will be broken.